gb-dbg/src/debugger/breakpoints/evaluation.rs

```diff
@@ -44,7 +44,9 @@ pub fn eval_binary_op(op: &Operator, lhs: u16, rhs: u16) -> u16 {
             }
         }
         Operator::LogicXor => {
-            if lhs != rhs {
+            if lhs == TRUE && rhs == FALSE {
+                TRUE
+            } else if lhs == FALSE && rhs == TRUE {
```

Comment:
```rust
let lhs_check = lhs != FALSE;
let rhs_check = rhs != FALSE;
if (lhs_check || rhs_check) && lhs_check != rhs_check {
```

Reply:
not sure it's a lot more readable I assume anyone reading this will be able to understand what Xor does